src/Components/DiaryCard.js

```diff
@@ -27,35 +27,22 @@ function DiaryCard({title, subtitle, description, color }) {
     const resultString = more ? description.slice(0, 100) + '...' : description;
 
     function ShowMore() {
-        setMore(false);
+        more ? setMore(false) : setMore(true)
```

Comment:
This is okay. But you can simplify this more like this `setMore(!more)`. Change this like this.